packages/api-client-core/src/exchanges/urlParamExchange.ts

```diff
@@ -1,6 +1,7 @@
+import type { Exchange } from "@urql/core";
 import { mapExchange } from "@urql/core";
 
-export const urlParamExchange = mapExchange({
+export const urlParamExchange: Exchange = mapExchange({
```

Comment:
Curious: why this one? I would have assumed `mapExchange` would be typed to return an `Exchange`.

Reply:
It was for a portable type thing -- typescript asked to put an explicit annotation to make it portable